### DISCUSSION TITLE
Figma - Prime Themes  - Prime Templates

### DISCUSSION THREAD

**Discussion by janeznavteh100** [primefaces/community#580] [Figma UI Kit] [answered]
Hi all, 
using PrimeNg for the short term.. 

What is the best option to work with PrimeNG and Figma? 
Is in paid template included fig file for that template? 
How to achieve those components in Figma match custom PrimeNg theme?

**Comment by onursenture** [comment, marked as answer]
Hi,

The best way to work with PrimeNG is to use the Lara theme and buy the [PrimeOne UI Kit](https://primeng.org/uikit). UI Kit, for now, only supports the Lara theme.

Paid templates have their own Figma files for their layout designs, but these Figma files don't include UI elements like buttons, input, etc. 

For custom themes, we recommend using Tokens Studio, which is introduced lately in our UI Kit.

**Comment by pavigitup** [comment]
I am using the PrimeNG Suite and have a license for the PrimeOne UI Kit, as well as purchased the PrimeNG Atlantis template. However, when creating a prototype in Figma for a page, the Figma kit of the template does not include any components. The components are part of the PrimeOne UI Kit Figma file. I can import the components from the PrimeOne UI Kit into the page for design. However, the look and feel of the components do not match the template. What am I missing? 


And what is the actual flow to use themes in primeone UI kit

**Reply by onursenture** [reply]
You haven’t missed anything. The PrimeOne UI Kit includes the Aura theme; however, not all templates were originally designed with Aura in mind. That said, we’ve been gradually updating the templates to improve Aura compatibility, although we’ve not yet fully transitioned everything. 

The same applies to PrimeNG Atlantis. The theme used in PrimeOne differs from the one in Atlantis, and there is no dedicated Figma file for the components used in Atlantis. The Atlantis Figma only includes layout-related assets such as the dashboard and landing page.
